site/source/docs/contributing/developers_guide.rst

```diff
@@ -158,7 +158,8 @@ If you change the layout of C structs or modify C defines that are used in
 JavaScript library files you may need to modify ``tools/struct_info.json``.  Any
 time that file is modified or a struct layout is changed you will need to run
 ``./tools/gen_struct_info.py`` to re-generate the information used by
-JavaScript.
+JavaScript. Note that you need to run both ``./tools/gen_struct_info.py`` and
+``./tools/gen_struct_info.py --wasm64``.
```

Comment:
@sbc100 Should we make `gen_struct_info` also do wasm64 automatically, rather than ask people to run the tool twice? Or do we need the option to only run one of the two?

Reply:
I don't think it really matters.. most folks should never need to run this anyway.. plus we have tests that fail if they forget.